### PR TITLE
feat: added files for landing page

### PR DESCRIPTION
**Pull request by Ranjeetk489**

### CODE REVIEW

File: css/content.css [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    margin: 6rem 6rem 6rem 8rem;
+    height: 50vh;
+  }
+  .main-content-title {
+    font-size: var(--font-size-1);
+    font-weight: var(--font-weight-1);
+    margin-bottom: 1rem;
+  }
+  .main-content-secondary {
+    /* padding: 0 0 0 1.4rem; */
```
Comment (DivyanshSareen) [inline css/content.css]: remove comment

File: css/css/content.css [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  padding: 0.7rem 1.1rem;
+  text-decoration: none;
+  font-size: var(--font-size-5);
+  font-weight: var(--font-weight-2);
+  color: var(--tertiary-color);
+  background-color: var(--complementary-color);
+}
+.main-content-btn:hover,
+.main-content-btn:focus {
+  box-shadow: 0px 12px 12px rgba(0, 0, 0, 0.25),
```
Comment (DivyanshSareen) [inline css/css/content.css:21]: can make a var for shadows if used in multiple files
`--var-name: 0px 12px 12px rgba(0, 0, 0, 0.25)`

File: css/css/content.css [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    margin: 6rem 6rem 6rem 8rem;
+    height: 50vh;
+  }
+  .main-content-title {
+    font-size: var(--font-size-1);
+    font-weight: var(--font-weight-1);
+    margin-bottom: 1rem;
+  }
+  .main-content-secondary {
+    /* padding: 0 0 0 1.4rem; */
```
Comment (DivyanshSareen) [inline css/css/content.css:114]: remove comment

File: css/css/style.css [hunk 4]
```diff
@@ -0,0 +1,15 @@
+@import url(variables.css);
+@import url(navbar.css);
+@import url(content.css);
+@import url(footer.css);
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  /* overflow: hidden; */
```
Comment (DivyanshSareen) [inline css/css/style.css:10]: remove commented code

**Review by DivyanshSareen — APPROVED** [review]
Nice site layout, good to merge

**Review by rutu14 — APPROVED** [review]

File: css/css/style.css (hunk 4 above)
Comment (Ranjeetk489) [reply css/css/style.css:10]: Made the said changes.. thanks for reviewing

**Pull request merged**